digitalaidseattle/plymouth-housing
# Inventory page

## Description

_Give a brief context and description of the code change. Highlight anything you want reviewers to pay attention to._
Created the basics of the inventory page. Light styling applied. Inventory page also connects with supabase database. Currently pulling manually made dummy data. Filters were also created for the inventory page which can be stacked with each other or used independently. The search filter is able to filter through all columns, not just the item. Pagination also created; page numbers and arrow icons are interactable. 

_The PR should have enough information so someone coming in fresh gets a sense of what's going on, but the details of the overall project can be documented on project.digitalaidseattle.org_

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

_Link user story from projects.digitalaidseattle.org_
https://projects.digitalaidseattle.org/project-details?recordId=recAoL7m4dISAVEX7
https://das-ph-inventory-tracker.atlassian.net/jira/core/projects/PIT/board?selectedIssue=PIT-53

- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

Ensure your local env file has the supabase url and anon key. Should populate on load. The inventory page should be able to handle filtering based on the dropdown menus or by search. The search filters based on all columns. 

![image](https://github.com/user-attachments/assets/d02d10f9-5ac7-42a9-b4e4-150793347023)
